kolibri/plugins/management/assets/src/views/class-edit-page/index.vue

```diff
@@ -78,7 +78,7 @@
 
           <!-- Logic for role tags -->
           <td class="table-cell table-role">
-            <user-role :role="user.kind" :omitLearner="true" />
+            <!-- <user-role :role="user.kind" :omitLearner="true" /> -->
```

Comment:
Yeah, I think we can clean it up when we stabilize the design for class coaches.